common/components/controllers/CreateProjectController.jsx

```diff
@@ -177,7 +187,16 @@ class CreateProjectController extends React.PureComponent<{||},State> {
   }
   
   onFormUpdate(formFields: {||}) {
-    this.setState({fieldsUpdated: true});
+    if (!this.state.clickedNext && !this.compareFormFields(this.state.currentFormFields, formFields)) {
```

Comment:
```suggestion
    if (!this.state.clickedNext && !_.isEqual(this.state.currentFormFields, formFields)) {
```
Did a little research, and it looks like _.isEqual will do the trick in lieu of our custom function.